### ISSUE TITLE
Crash on startup

### ISSUE THREAD

**Issue by migonin** [sublimehq/sublime_merge#332]
**Version info**

 - OS: Mac OS X 10.14.2
 - Build: 1092, 1097, 1098
 - Macbook Pro 2018 15", M942LL/A

**Description**

App crashes on startup.

**Steps to reproduce the behavior:**
1. Download build
2. Unzip it
3. (optional) Move to Applications folder
4. Run it

**Expected behavior**

App runs

**Actual behavior**

App crashes

**Comment by srbs** [comment]
@migonin, did you create an account or setup a new Mac and (without logging out or restarting) _then_ run Sublime Merge and it crash?

I ran into a similar issue migrating to a new Mac. It ended up just being a weird folder permission state that was solved by just letting the OS refresh itself after the initial login process.

**Comment by migonin** [comment]
@srbs yeah it was first steps on my new Macbook. Looks like Apple bug, other apps like iTunes had problems also.

**Comment by srbs** [comment]
@migonin, feel free to close the issue if you'd like. Otherwise, it's up to the devs if they felt this is worth notifying the users to restart.
